pkg/cmd/server/bootstrappolicy/all.go

```diff
@@ -1,11 +1,20 @@
 package bootstrappolicy
 
 import (
-	rbacrest "k8s.io/kubernetes/pkg/registry/rbac/rest"
+	"k8s.io/kubernetes/pkg/apis/rbac"
```

Comment:
@juanvallejo  figure out what in oc requires this so we can decide what to do.  It's weird that we got busted here.